packages/contracts/docs/osx/02-how-to-guides/01-dao/01-best-practices.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Best Practices
+---
+
+## Advices for Operating your DAO
```

Comment:
Doesn't the best practices section allow for more? I think the brainstorm had more topics, maybe they are elsewhere